components/button/index.css

```diff
@@ -110,11 +145,11 @@ governing permissions and limitations under the License.
     padding-inline-end: 0;
   }
 
-  --spectrum-button-focus-ring-color: var(--spectrum-button-m-primary-fill-texticon-focus-ring-color-key-focus);
+  /* --spectrum-button-focus-ring-color: var(--spectrum-button-m-primary-fill-texticon-focus-ring-color-key-focus); */
```

Comment:
Are we able to remove this commented out line now?

Reply:
yes, we get a good focus ring color on all button variants.